src/Adapter/Product/SpecificPrice/Update/SpecificPricePriorityUpdater.php

```diff
@@ -52,12 +82,12 @@ public function setPrioritiesForProduct(ProductId $productId, PriorityList $prio
     {
         try {
             if (!SpecificPrice::setSpecificPriority($productId->getValue(), $priorityList->getPriorities())) {
-                throw new CannotSetSpecificPricePrioritiesException(sprintf(
```

Comment:
Seems like this `CannotSetSpecificPricePrioritiesException` is no longer used maybe it should be cleaned?